integration-tests/src/test/java/com/linkedin/kafka/clients/largemessage/LargeMessageIntegrationTest.java

```diff
@@ -28,8 +31,8 @@
 import java.util.Properties;
 import java.util.Set;
 
-import static org.testng.Assert.assertEquals;
-import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.*;
```

Comment:
do we like star imports ?

Reply:
Nope. This is just intellij messing with me.